userservice/src/main/java/com/infodation/userservice/config/SecurityConfig.java

```diff
@@ -11,11 +12,17 @@ public class SecurityConfig {
     @Bean
     public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
         http
-                .authorizeHttpRequests(authorize -> authorize
-                .antMatchers("/hello").permitAll()
+                .csrf().disable()
+                .authorizeRequests()
+                .antMatchers(HttpMethod.GET, "/api/users").permitAll()
+                .antMatchers(HttpMethod.GET, "/api/users/**").permitAll()
+                .antMatchers(HttpMethod.POST, "/api/users").permitAll()
+                .antMatchers(HttpMethod.PUT, "/api/users/**").permitAll()
+                .antMatchers(HttpMethod.DELETE, "/api/users/**").permitAll()
```

Comment:
You can shorten these code by
```suggestion
       .antMatchers("/api/users/**").permitAll()
```